templates/scripts/add-vpn-user.sh

```diff
@@ -65,11 +85,12 @@ cat <<-EOF > ${CONFIG_FILE}
 PrivateKey = $client_private_key
 ListenPort = 34567
 Address = $next_ip/32
+DNS = $dns_server
 
 [Peer]
 # VPN server side
 PublicKey = $server_public_key
-AllowedIPs = 10.10.0.0/16
+AllowedIPs = $vpc_cidr, $dns_server/32
```

Comment:
Similar here, why would I want to route a request that should be on my local network through the vpn?

Reply:
as confirmed, that $dns_server is retrieved from vpn pod (not from local)